index.html

```diff
@@ -120,9 +127,7 @@ <h5 class=" h5 ">Historia del Atletismo</h5>
 
     </footer>
     <!-- Footer -->
```

Comment:
Remove this comment, it's already declared above no need to state it again.

Reply:
done